src/Sylius/Bundle/PaymentBundle/Provider/PaymentNotifyProviderInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\PaymentBundle\Provider;
+
+use Sylius\Component\Payment\Model\PaymentInterface;
+use Sylius\Component\Payment\Model\PaymentMethodInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+interface PaymentNotifyProviderInterface
```

Comment:
```suggestion
interface NotifyPaymentProviderInterface
```
```suggestion
interface NotifiedPaymentProviderInterface
```
or smth alike, the current name sounds like I'd put a payment inside and get a notification as the result